PORTAL/jobs/_pyperformance.py

```diff
@@ -1848,7 +2108,9 @@ def as_rendered_row(self, columns):
                 rendered = self.baseline or ''
             elif column == 'mean':
                 if self.isbaseline:
-                    rendered = self.BASELINE_REF
+                    # TYPE: BROKEN
+                    # rendered = self.BASELINE_REF
```

Comment:
What's broken here?

Reply:
`BASELINE_REF` doesn't appear anywhere in the source as far as I can tell.

Reply:
I expect it was supposed to be `PyperfComparisonValue.BASELINE` and I missed it when refactoring at some point.